CHANGES.md

```diff
@@ -20,6 +20,11 @@ Synapse 1.32.0 (2021-04-20)
 that can overwhelm connected Prometheus instances. This issue was not present in
 1.32.0rc1, and is fixed in 1.32.1. See the changelog for 1.32.1 above for more information.
 
+**Note:** This release also introduces a change that may affected Synapse modules that
```

Comment:
Might be worth saying "This release *mistakenly* included a change that..." or something to that effect?

Reply:
Heh, good point.